src/sentry/incidents/endpoints/serializers/workflow_engine_incident.py

```diff
@@ -104,21 +236,25 @@ def serialize(
         **kwargs,
     ) -> IncidentSerializerResponse:
         """
-        Temporary serializer to take an OpenPeriod and serialize it for the old metric alert rule endpoints
+        Temporary serializer to take a GroupOpenPeriod and serialize it for the old incident endpoint
         """
-
+        incident_group_open_period = IncidentGroupOpenPeriod.objects.get(group_open_period=obj)
         date_closed = obj.date_ended.replace(second=0, microsecond=0) if obj.date_ended else None
         return {
-            "id": str(obj.id),
-            "identifier": str(obj.id),
+            "id": str(incident_group_open_period.incident_id),
+            "identifier": str(
+                obj.id
+            ),  # TODO this isn't the same thing, it's Incident.identifier which we might want to add to IncidentGroupOpenPeriod
             "organizationId": str(obj.project.organization.id),
-            "projects": [obj.project.slug],
+            "projects": attrs["projects"],
             "alertRule": attrs["alert_rule"],
             "activities": attrs["activities"] if "activities" in self.expand else None,
-            "status": self.get_incident_status(obj.group.priority),
-            "statusMethod": IncidentStatusMethod.RULE_TRIGGERED.value,  # We don't allow manual updates or status updates based on detector config updates
+            "status": self.get_incident_status(
+                obj.group.priority
+            ),  # TODO could be closed, need to handle
+            "statusMethod": IncidentStatusMethod.RULE_TRIGGERED.value,  # TODO manual isn't allowed. could be RULE_UPDATED if status is closed
             "type": IncidentType.ALERT_TRIGGERED.value,  # IncidentType.Detected isn't used anymore
-            "title": obj.group.title,
+            "title": obj.group.title,  # TODO this corresponds to the detector name / alert rule name, is the openperiod the same?
```

Comment:
> is the openperiod the same?

The open period doesn't have a notion of title, but the group's title should be the alert rule name/something similar